### PR TITLE
Make bad xlsx error better

### PR DESCRIPTION
**Pull request by kindly**

### CODE REVIEW

File: flattentool/input.py [hunk 1]
```diff
@@ -555,7 +556,10 @@ def get_sheet_lines(self, sheet_name):
 
 class XLSXInput(SpreadsheetInput):
     def read_sheets(self):
-        self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
+        try:
+            self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
+        except zipfile.BadZipFile as e:
+            raise Exception("The supplied file has extension .xlsx but isn't a XSLX file.")
```
Comment (robredpath) [inline flattentool/input.py]: This should read "isn't *an* XLSX file" (change 'a' to 'an' and correct order of XLSX)

File: flattentool/input.py [hunk 2]
```diff
@@ -555,7 +560,10 @@ def get_sheet_lines(self, sheet_name):
 
 class XLSXInput(SpreadsheetInput):
     def read_sheets(self):
-        self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
+        try:
+            self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
+        except BadZipFile as e:
+            raise Exception("The supplied file has extension .xlsx but isn't an XLSX file.")
```
Comment (Bjwebb) [inline flattentool/input.py]: Can we use a custom exception type here, to make it possible to specifically catch this in CoVE without needing to ammend flatten-tool further?

**Comment by robredpath** [comment]
[BadZipFile error message is confusing](https://trello.com/c/UYvGZIZ7/27-badzipfile-error-message-is-confusing)

**Comment by odscjames** [comment]
Should this have had  CHANGELOG entries?

**Comment by odscjames** [comment]
Done in different p.r.

**Pull request merged**